tests/installation/playwright-should-work-with-relative-home-path.spec.ts

```diff
@@ -14,8 +14,11 @@
  * limitations under the License.
  */
 import { test } from './npmTest';
+import os from 'os';
 
 test('playwright should work with relative home path', async ({ exec }) => {
+  test.skip(os.platform().startsWith('win'));
```

Comment:
This is skipped on windows, since I don't think HOME dir applies, but if it doesn, feel free to fix!